FileTypeSupport.MXLIFF/MXLIFFWriter.cs

```diff
@@ -33,109 +32,107 @@ public void Dispose()
 
         public void FileComplete()
         {
-            using (XmlTextWriter wr = new XmlTextWriter(nativeFileProperties.OutputFilePath, Encoding.UTF8))
+            using (var wr = new XmlTextWriter(_nativeFileProperties.OutputFilePath, Encoding.UTF8))
             {
                 wr.Formatting = Formatting.None;
-                targetFile.Save(wr);
-                targetFile = null;
+                _targetFile.Save(wr);
+                _targetFile = null;
             }
         }
 
         public void GetProposedOutputFileInfo(IPersistentFileConversionProperties fileProperties, IOutputFileInfo proposedFileInfo)
         {
-            originalFileProperties = fileProperties;
         }
 
         public void Initialize(IDocumentProperties documentInfo)
         {
-            textExtractor = new MXLIFFTextExtractor();
+	        _documentProperties = documentInfo;
+            _textExtractor = new MXLIFFTextExtractor();
         }
 
         public void ProcessParagraphUnit(IParagraphUnit paragraphUnit)
         {
-            string id = paragraphUnit.Properties.Contexts.Contexts[0].GetMetaData("ID");
-            XmlNode xmlUnit = targetFile.SelectSingleNode("//x:trans-unit[@id='" + id + "']", nsmgr);
+            var id = paragraphUnit.Properties.Contexts.Contexts[0].GetMetaData("ID");
+            var xmlUnit = _targetFile.SelectSingleNode("//x:trans-unit[@id='" + id + "']", _nsmgr);
```

Comment:
check for null validation on _targetFile

Reply:
It doessn't need to is always instantiated in line 63.